av1an/src/main.rs

```diff
@@ -121,6 +121,10 @@ pub struct CliOpts {
   #[clap(short, required = true)]
   pub input: Vec<PathBuf>,
 
+  /// Alternate input for scene detection (must have the same frame count)
```

Comment:
What happens when the two inputs don't have the same frame count?

Reply:
> What happens when the two inputs don't have the same frame count?

This needs a check, With at least a warning to prevent future hair pulling.

Reply:
This is indeed something that should be added to prevent... annoying reports in the future.